handlers/dashboard.go

```diff
@@ -277,14 +283,23 @@ func getNextWithdrawalRow(queryValidators []uint64) ([][]interface{}, error) {
 		withdrawalCredentialsTemplate = `<span class="text-muted">N/A</span>`
 	}
 
+	var withdrawalAmount uint64
+	if nextValidator.WithdrawableEpoch <= epoch {
+		// full withdrawal
+		withdrawalAmount = nextValidator.Balance
+	} else {
+		// partial withdrawal
+		withdrawalAmount = nextValidator.Balance - utils.Config.Chain.Config.MaxEffectiveBalance
+	}
+
 	nextData := make([][]interface{}, 0, 1)
 	nextData = append(nextData, []interface{}{
 		template.HTML(fmt.Sprintf("%v", utils.FormatValidator(nextValidator.Index))),
 		template.HTML(fmt.Sprintf(`<span class="text-muted">~ %s</span>`, utils.FormatEpoch(uint64(utils.TimeToEpoch(timeToWithdrawal))))),
 		template.HTML(fmt.Sprintf(`<span class="text-muted">~ %s</span>`, utils.FormatBlockSlot(utils.TimeToSlot(uint64(timeToWithdrawal.Unix()))))),
 		template.HTML(fmt.Sprintf(`<span class="">~ %s</span>`, utils.FormatTimeFromNow(timeToWithdrawal))),
 		withdrawalCredentialsTemplate,
-		template.HTML(fmt.Sprintf(`<span class="text-muted">~ %s</span>`, utils.FormatAmount(new(big.Int).Mul(new(big.Int).SetUint64(nextValidator.Balance-utils.Config.Chain.Config.MaxEffectiveBalance), big.NewInt(1e9)), "ETH", 6))),
+		template.HTML(fmt.Sprintf(`<span class="text-muted">~ %s</span>`, utils.FormatAmount(new(big.Int).Mul(new(big.Int).SetUint64(withdrawalAmount), big.NewInt(1e9)), "ETH", 6))),
```

Comment:
**Unrelated to the bugfix:** Here in the dashboard we show a `~` for the amount but we don't do this for the validator view. Should this be consistent?

Reply:
Good catch, has already been fixed here though: https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2010/files#diff-3ebb39e6ec61a163864798ea1e5af8c31d00e716a2b6cd1625c12336ef8dd281L289